gunicorn.conf.py

```diff
@@ -1,14 +1,24 @@
 import os
 DEPLOYMENT_MODE = os.environ.get('DEPLOYMENT_MODE')
 
+worker_class = 'gevent'
+workers = 4
```

Comment:
There is a relationship between the number of workers, the connection pool, and how many connections end up getting opened. It's a hard relationship to express explicitly, and it takes a long time to test changes, and we don't have great ways of measuring whether certain settings are 'better' or not, but feel free to mess around.